docs/scos/dev/guidelines/performance-guidelines/spryker-elastic-computing/spryker-elastic-computing.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Spryker Elastic Computing
+description: This guideline explains how to use Elastic Computing functionality.
+last_updated: May 16, 2022
+template: concept-topic-template
+---
+
+Spryker provides several elastic computing features OOTB that allows to utilise computational resources in a more efficient way. 
+
+# Monitoring
+For enabling New Relic monitoring for P&S and queue workers infrastructure, please follow the next steps:
+
+1. Update `spryker/monitoring` module at least to v2.5.0.
+2. Add `queue:task:start` command to the argument grouped transactions in `Pyz\Zed\Monitoring\MonitoringConfig` configuration class
+```php
+namespace Pyz\Zed\Monitoring;
+
+use Spryker\Zed\Monitoring\MonitoringConfig as BaseMonitoringConfig;
+
+class MonitoringConfig extends BaseMonitoringConfig
+{
+    /**
+     * @api
+     *
+     * @return array<string>
+     */
+    public function getGroupedByArgumentTransactions(): array
+    {
+        return ['queue:task:start'];
+    }
+}
+```
+3. Add to the `Pyz\Zed\Monitoring\Business\MonitoringBusinessFactory` following lines of code:
+```php
+public function getMonitoringTransactionNamingStrategies(): array
+    {
+        return [
+            return new FirstArgumentMonitoringConsoleTransactionNamingStrategy($this->getConfig()->getGroupedByArgumentTransactions());
+        ];
+    }
+```
+This will aggregates all different queue worker transactions under corresponding `queue:task:start [queue_name]` transaction name in New Relic.
+
+As a result, the transactions will appear in New Relic dashboard as on the following illustration:
+
+# RAM-aware batch processing integration
+Batch processing allows significantly speed up processing operations. Moving more data into RAM at once allows decrease the number of I/O thus winning some operations wall time. To win even more time one can calculate batch size dynamically, based on available RAM. 
+
+This can be achieved with elastic batch concept introduced in `spryker/data-import` module. The following instruction will explain the integration details of elastic batch using the glossary data import as an example.
```

Comment:
As I understand, the module is responsible for data import in general. Did you just add batch processing to the `spryker/data-import`?